test/value-slider-view-test.ts

```diff
@@ -38,6 +43,14 @@ describe("The Value Slider", () => {
     });
   };
 
+  // fake the event listeners for enzyme
```

Comment:
Just curious:  What is the story with this?

Reply:
Enzyme doesn't support `window.addEventListener()` and its only used to check for resizing which the test doesn't exercise.  The docs say to just stub it out.